### PR TITLE
Cleanup identifiers usage in /assets/all

### PR DESCRIPTION
**Pull request by nebolax**
Updates the docs, removes unneeded code and makes sure that if `identifiers` list is provided, it has the highest priority in `POST /assets/all` enpoint

### CODE REVIEW

File: rotkehlchen/api/v1/schemas.py [hunk 1]
```diff
@@ -2042,11 +2042,10 @@ def make_assets_post_query(
             elif data['ignored_assets_handling'] == IgnoredAssetsHandling.SHOW_ONLY:
                 ignored_assets_filter_params = ('IN', self.db.get_ignored_asset_ids(cursor))
 
-            if data['show_user_owned_assets_only'] is True:
+            identifiers = data['identifiers']
+            if data['show_user_owned_assets_only'] is True and identifiers is None:
                 globaldb_read_cursor.execute('SELECT asset_id FROM user_owned_assets;')
                 identifiers = [entry[0] for entry in globaldb_read_cursor]
-            else:
-                identifiers = data['identifiers']
```
Comment (yabirgb) [inline rotkehlchen/api/v1/schemas.py:2049]: why this change?

File: rotkehlchen/api/rest.py [hunk 2]
```diff
@@ -1270,15 +1266,6 @@ def query_list_of_all_assets(
...
             )
 
-        if identifiers is not None and len(identifiers) != assets_found:
-            not_found = []
-            found_assets = {asset['identifier'] for asset in assets}
-            for asset in identifiers:
-                if asset not in found_assets:
-                    not_found.append(asset)
-            msg = f"Queried identifiers {','.join(not_found)} are not present in the database"
-            return api_response(wrap_in_fail_result(msg), status_code=HTTPStatus.CONFLICT)
```
Comment (yabirgb) [inline rotkehlchen/api/rest.py:1280]: then you need to update the docs but why this change?

**Review by yabirgb — COMMENTED** [review]
I'm not sure about the background for the changes in this PR. If you could explain would be nice

File: rotkehlchen/api/rest.py (hunk 2 above)
Comment (nebolax) [reply rotkehlchen/api/rest.py:1280]: This part of the code looked a bit strange to me. I asked Luki and he said that this check is not used / relied upon anywhere anyway, so I removed the unused code.

File: rotkehlchen/api/v1/schemas.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/api/v1/schemas.py:2049]: Yes, that is what I would expect from the condition you wrote

File: rotkehlchen/tests/api/test_assets.py [hunk 3]
```diff
@@ -470,22 +470,6 @@ def test_get_all_assets(rotkehlchen_api_server):
...
     assert result['entries'][2]['identifier'] == A_BTC
 
-    # ask for a non existent asset
-    response = requests.post(
-        api_url_for(
-            rotkehlchen_api_server,
-            'allassetsresource',
-        ),
-        json={
-            'identifiers': ['my_life'],
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_assets.py:480]: Hahahah @yabirgb nicely done. Too bad @nebolax is removing it

**Review by LefterisJP — APPROVED** [review]
LGTM. Error is just blockchain.info timeout
